extensions/hibernate-reactive/deployment/src/test/java/io/quarkus/hibernate/reactive/sql_load_script/FileNotFoundSqlLoadScriptTestCase.java

```diff
@@ -22,7 +22,7 @@ public class FileNotFoundSqlLoadScriptTestCase {
             .assertException(t -> assertThat(t)
                     .isInstanceOf(ConfigurationException.class)
                     .hasMessageContainingAll("Unable to find file referenced in '"
-                            + "quarkus.hibernate-orm.sql-load-script=file-that-does-not-exist.sql'. Remove property or add file to your path."));
+                            + "quarkus.hibernate-orm.\"<default-reactive>\".sql-load-script=file-that-does-not-exist.sql'. Remove property or add file to your path."));
```

Comment:
That's what I was mentioning in my previous review when I mentioned the "config name". The new expecation is just wrong, we really do want `quarkus.hibernate-orm.sql-load-script` here.

Reply:
Fixed the assertion